### PR TITLE
Greek translation from Crowdin.

### PR DESCRIPTION
**Pull request by magul**
Made by:
* Chara Lagou (https://crowdin.com/profile/lagou.chara)
* Eleni Lixourioti (https://crowdin.com/profile/inelegr88)
* Frosso Papanastasiou (https://crowdin.com/profile/efpapanastasiou)
* konstadina89fifli (https://crowdin.com/profile/konstadina89fifli)
* Kristi Achilleos (https://crowdin.com/profile/Krispy2009)
* Nick Mavrakis (https://crowdin.com/profile/manikos)
* Panagiota Bilianou (https://crowdin.com/profile/pbilianou)
* rorogr (https://crowdin.com/profile/rorogr)
* sotsi (https://crowdin.com/profile/sotsi)
* Stavros Anastasiadis (https://crowdin.com/profile/kickapoo)
* vpapg (https://crowdin.com/profile/vpapg)

### CODE REVIEW

File: el/django_templates/README.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+Βλέπετε, στην HTML δεν μπορείτε να γράψετε κώδικα σε Python επειδή οι browsers δεν καταλαβαίνουν αυτή τη γλώσσα. Εκείνοι ξέρουν μόνο HTML, CSS και Javascript. Ξέρουμε ότι η HTML είναι στατική γλώσσα ενώ η Python περισσότερο δυναμική.
+
+Τα **Django template tags** μας επιτρέπουν να μεταφέρουμε πράγματα της Python μέσα στην HTML, ούτως ώστε να χτίσετε δυναμικές ιστοσελίδες γρηγορότερα. Τέλεια!
+
+## Template προβολή λίστας των post
+
+Σε προηγούμενο κεφάλαιο δώσαμε στο template μας μια λίστα από posts μέσα σε μια μεταβλητή με το όνομα `posts`. Τώρα θα την προβάλουμε στην HTML. Άλλωστε αυτό θέλαμε να κάνουμε από την αρχή.
+
+Για να εκτυπώσουμε μια μεταβλητή μέσα στα Django templates, χρησιμοποιούμε διπλές αγκύλες ({{ }}) με το όνομα της μεταβλητής ανάμεσα:
```
Comment (magul) [inline el/django_templates/README.md]: @manikos what is this `({{ }})` here for?

Comment (manikos) [reply el/django_templates/README.md]: In Greek language we describe `[`, `]`, `{` and `}` as one word (αγκύλη == brace). So, in order not to confuse them I wrote it inside a parenthesis. As I can see the build fails. Should I remove it?

Comment (magul) [reply el/django_templates/README.md]: `{% raw %}` most probably should do the trick here. Generally it should not fail. You can also try to build it locally (using node and gitbook).

And, please, apply change on Crowdin (so I will be able to rebuild this pull request).

Comment (manikos) [reply el/django_templates/README.md]: I'll do this right now.

Comment (magul) [reply el/django_templates/README.md]: Great!

Comment (manikos) [reply el/django_templates/README.md]: Done!

File: el/django_forms/README.md [hunk 2]
```diff
@@ -0,0 +1,489 @@
...
+
+Συγχαρητήρια! Η εφαρμογή σας γίνεται όλο και πιο πλήρης!
+
+Εάν χρειάζεστε περισσότερες πληροφορίες για τις φόρμες Django, θα πρέπει να διαβάσετε το: https://docs.djangoproject.com/en/2.0/topics/forms/
+
+## Ασφάλεια
+
+Το να μπορείς να δημιουργήσεις νέες δημοσιεύσεις κάνοντας απλώς ένα κλικ σε ένα σύνδεσμο είναι φοβερό! Αλλά τώρα, οποιοσδήποτε επισκεφθεί την σελίδα σας θα μπορεί να φτιάξει μία καινούρια δημοσίευση blog, και αυτό είναι κάτι που πιθανώς δεν θέλετε. Ας το κάνουμε έτσι ώστε το κουμπί να εμφανίζεται μόνο για εσάς και για κανέναν άλλο.
+
+Ανοίξτε το αρχείο `Blog/templates/blog/base.html` και βρείτε το `page-header` `div` καθώς και το tag του link (<a>). Θα μοιάζει κάπως έτσι:</p>
```
[the remark this answers is not in the document]
Comment (manikos) [reply el/django_forms/README.md]: Oh crap. I guess I'll have to build it locally and then post back here (or message you via crowdin). Is that ok?

[the remark this answers is not in the document]
Comment (magul) [reply el/django_forms/README.md]: That looks like the last problem here. You can try to build it on Crowdin (I don't know if non-admin can do that).

[the remark this answers is not in the document]
Comment (manikos) [reply el/django_forms/README.md]: OK done. Last build (?)!

**Comment by magul** [comment]
@manikos It built successfully. Current version is available under: https://magul.gitbooks.io/django-girls-tutorial/content/v/32921cb728553cdfb0b511720664540108f59c37/

You're welcome to review it (but I still prefer wait at least a week for maybe second reviewer, as You're primary contributor here).

**Comment by manikos** [comment]
Wow! That looks great! Of course I'll wait for a second reviewer. No problem. Thank you for the PR!

**Comment by magul** [comment]
Ok, one week passed and there's no issues reported, so I'm going to merge it. BTW @manikos there's already changes in English version, so You can update Crowdin translation (and I will create yet another pull request to update it on GitHub).

**Comment by manikos** [comment]
Sure. I'm on it ;)

**Pull request merged**